rafs/src/metadata/direct_v6.rs

```diff
@@ -756,37 +757,63 @@ impl RafsInode for OndiskInodeWrapper {
         let blocks_count = div_round_up(inode.size(), EROFS_BLOCK_SIZE);
         let mut target: Option<u64> = None;
 
-        let nid = 'outer: for i in 0..blocks_count {
-            let head_entry = self.get_entry(i as usize, 0).map_err(err_invalidate_data)?;
-            // `e_nameoff` is offset from each single block?
+        // find target block
```

Comment:
How about extracting this as a helper function?
And seems it may simplify by using usize instead of i32?

Reply:
Done.